### DISCUSSION TITLE
A negative test

### DISCUSSION THREAD

**Discussion by doquang** [idaholab/moose#23759] [Q&A Tools] [answered]
Hi,

Is there any way I can determine a negative test for the TestHarness? I mean, with a particular input file, the simulation will fail. And the test will say OK, if this simulation fails.

Thanks a lot,
/ Minh

**Comment by GiudGiud** [comment, marked as answer]
Yes it's the runexception test

**Reply by GiudGiud** [reply]
https://mooseframework.inl.gov/python/testers/RunException.html

**Reply by doquang** [reply]
Thank you, @GiudGiud
